CHANGELOG.md

```diff
@@ -21,7 +21,7 @@ New features:
 * `equal?` on foreign objects will check if the underlying objects are equal
   if both are Java interop objects.
 
-* `remove` on foreign objects will send `REMOVE`, `size` will send `GET_SIZE`,
+* `delete` on foreign objects will send `REMOVE`, `size` will send `GET_SIZE`,
   and `keys` will send `KEYS`. `respond_to?(:size)` will send `HAS_SIZE`,
```

Comment:
doc/contributor/interop.md says `respond_to?(:to_a)`/`respond_to?(:to_ary)` send `HAS_SIZE`, which one is it?
All 3 of them?
IMHO it makes more sense for the user with `respond_to?(:size)` since `size` will send `GET_SIZE`.
`to_a/to_ary` is just convenience, isn't it?

Reply:
> doc/contributor/interop.md says respond_to?(:to_a)/respond_to?(:to_ary) send HAS_SIZE, which one is it?

I don't know what you mean. `respond_to?(:to_a)`/`respond_to?(:to_ary)` sends `HAS_SIZE`. `size` sends `GET_SIZE`. `respond_to?(:size)` sends `HAS_SIZE`. There's no contradiction is there?

> `to_a`/`to_ary` is just convenience, isn't it?

Well I mean it's all convenience. It's all there to allow you to somewhat naturally use foreign objects as if they were Ruby objects.

Reply:
I think doc/contributor/interop.md should mention all 3 in each place any of the 3 is mentioned, otherwise it's hard to make the connection and people might find the respond_to?(:to_a) before even though respond_to?(:size) is more idiomatic.

Reply:
I added to 'What messages are sent for Ruby syntax on foreign objects', but not 'How to send messages using idiomatic Ruby', because `respond_to?(:to_a)` isn't the idiomatic way to send `HAS_SIZE` - `respond_to?(:size)` is.

Reply:
OK, sounds good.